tests/e2e/run-scanner-v4.sh

```diff
@@ -9,6 +9,11 @@ set -euo pipefail
 REPORTS_DIR=$(mktemp -d)
 FAILED=0
 
+echo "Worker node types for Scanner V4 tests:"
+kubectl get nodes -o json | \
```

Comment:
Could you print an example of the output?

Reply:
Here:
```
Worker node types for Scanner V4 tests:
rox-ci-67222272-l64l5-master-0.c.acs-san-stackroxci.internal: n2-standard-4
rox-ci-67222272-l64l5-master-1.c.acs-san-stackroxci.internal: n2-standard-4
rox-ci-67222272-l64l5-master-2.c.acs-san-stackroxci.internal: n2-standard-4
rox-ci-67222272-l64l5-worker-a-mxjmq: e2-standard-16
rox-ci-67222272-l64l5-worker-b-hllvq: e2-standard-16
```

Actually I should have left out the work "Worker", but if you don't mind I will fix this in a follow-up PR, as I would say this change isn't worth the costs of a CI run.